src/Simplex/Chat/Store/Profiles.hs

```diff
@@ -120,6 +121,11 @@ createUserRecordAt db (AgentUserId auId) Profile {displayName, fullName, image,
       (profileId, displayName, userId, True, currentTs, currentTs, currentTs)
     contactId <- insertedRowId db
     DB.execute db "UPDATE users SET contact_id = ? WHERE user_id = ?" (contactId, userId)
+    DB.execute
```

Comment:
as this isn't a direct part of user record, it's better to create separately in CreateActiveUser and getCreateActiveUser, same as `createContact db user simplexContactProfile` in the former.